jekyll/_cci2/pipelines.adoc

```diff
@@ -12,48 +11,63 @@ contentTags:
 
-== Dashboard
-When visiting your CircleCI dashboard, you are shown a list of recently-triggered pipelines for your organization.
+Pipelines can be triggered in the following ways:
 
-image::/docs/assets/img/docs/pipelines-dashboard.png[Screenshot of the pipelines dashboard in the CircleCI app]
+* Automatically: code changes, scheduled events, custom webhooks
+* Manually: CircleCI app or API
 
-[#pipeline-architecture]
-== Pipeline architecture
+Pipelines provide a flexible framework for creating conditional workflows based on xref:workflows#using-filters-in-your-workflows[filters], xref:workflows#workflows-configuration-examples[job requirements], and xref:selecting-a-workflow-to-run-using-pipeline-parameters#[pipeline parameters], allowing you to customize your CI/CD process to fit your project's needs.
 
-A pipeline is composed of workflows, which are composed of jobs. By navigating from a pipeline to a specific job, you can access your job output, test results, and artifacts through several tabs.
+== Pipelines dashboard
+Your pipelines dashboard displays recently-triggered pipelines for your organization. Use the various filters to find the pipelines you need. Click through to view workflow maps and individual job steps. Access quick controls to rerun and cancel workflows
 
-image::/docs/assets/img/docs/pipelines-job-step-test-artifact.png[Job tab options in the CircleCI web app]
+image::pipelines-dashboard.png[Screenshot of the pipelines dashboard in the CircleCI app]
 
-The output of each job can be opened in a new tab (in either raw or formatted styling) with a unique link, making it shareable between team members.
+From the pipelines dashboard you can find the _pipeline ID_. The pipeline ID is a unique ID for a pipeline that has been triggered. You can use the pipeline ID to access pipeline information via the link:https://circleci.com/docs/api/v2/index.html#tag/Pipeline/operation/getPipelineValuesById[API].
 
-image::/docs/assets/img/docs/pipelines-job-output.png[Download and share job step output]
+image::pipeline-id.png[Screenshot showing the location on the pipeline ID]
 
 == Pipelines and triggers
 
-Set up pipelines and triggers to define when and how pipelines are automatically triggered for a project. These triggers may be from external services (custom webhooks), or from commits or tags on a version control system.
+NOTE: The pipeline and trigger features available to you depend on how your code is integrated and where it is stored. For a feature availability overview, see the xref:version-control-system-integration-overview#[VCS integration overview] page.
+
+NOTE: **Using CircleCI server?** The Pipeline and Trigger pages under Project Settings and pipeline and trigger add/edit functionality is **not** available.
+
+Set up pipelines to define the code to checkout and the config to use for your project CI/CD..
```

Comment:
```suggestion
Set up pipelines to define the code to checkout and the config to use for your project CI/CD.
```

Reply:
Actually, reading this again in context, I wonder if talking about checkout and config here is too soon - I suppose users don't set up pipelines **in order to** define config and checkout source.

Maybe something like
> Set up pipelines to define the different CI/CD [units/pieces/scripts/?] that make up your project.

Reply:
how about: "Set up pipelines to define the various CI/CD "units" that make up your project."

Reply:
i like it